packages/create-ima-app/template/jest.config.js

```diff
@@ -4,11 +4,13 @@ module.exports = {
   modulePaths: ['<rootDir>/'],
   setupFiles: ['@ima/core/setupJest.js', '<rootDir>/setupJest.js'],
   snapshotSerializers: ['enzyme-to-json/serializer'],
-  testRegex: '(/__tests__/).*Spec\\.jsx?$',
+  testRegex: '(/__tests__/).*(Spec|Integration)\\.jsx?$',
```

Comment:
I don't think we need to define new file naming rules for integration tests, since we are running them as part of unit tests. So we don't have to name the test files `*Integration.js`, but we can use`*Spec.js` also for integration test.

We already have some sort of integration tests in `@ima/core`, where we also use `*Spec.js` convention, so we will be treating them the same here.